storage/blob.go

```diff
@@ -32,17 +61,23 @@ type BlobReader interface {
 	io.Closer
 	// Size returns the size of the blob
 	Size() (int64, error)
+	// GetStorageFormatVersion returns the storage format version associated with the blob.
+	GetStorageFormatVersion() FormatVersion
```

Comment:
nit: the `Get` feels unnecessary to me, and isn't parallel with the `Size()` call, which is not `GetSize()`. I'd prefer we keep them parallel.

Reply:
To clarify, you mean you'd prefer that the method be named `StorageFormatVersion()`?

Reply:
Yes, that is what I meant.

Reply:
I would just use `Version` or `Format`. in a similar way how `StorageSizeBytes` would feel redundant.

Reply:
We use "Version" for too many things already. (Try `git grep -w Version`!) `Format` is less overused, but is potentially a lie: these aren't necessarily just changes to file format. I named the concept of "storage format version" very deliberately in order to avoid any misleading senses or ambiguity.

Reply:
Yeah, it's a lie, but is it a good lie :D?

Reply:
You suggested `Scheme` elsewhere, and I can't find it now, but I like that in place of `Format`. But I'd still want to qualify it as "storage scheme version" or `storage.SchemeVersion` instead of "storage format version" or `storage.FormatVersion`.

I started actually doing this, but it adds a LOT of diff chunks and would really complicate reviewing of this PR. I suggest we keep FormatVersion, unless it's absolutely unacceptable to anyone.

Reply:
For a rename we can follow up with a separate PR, renames are easy to do automatically with Goland and easy to review.